ignite/metrics/rouge.py.orig

```diff
@@ -0,0 +1,152 @@
+from ignite.metrics import Metric
```

Comment:
Please do not commit useless files

Reply:
Sorry, I had formatting issue on my side and generated this file while trying to fix that. was not aware of it.